src/main/java/org/mabb/fontverter/opentype/OpenTypeFont.java

```diff
@@ -139,9 +140,34 @@ public void normalize() throws IOException {
         if (getOs2() == null)
             createNewOS2WinMetricsTable();
 
-        if (getNameTable() == null)
+        if (getNameTable() == null) {
             setName(NameTable.createDefaultTable());
+        } else {
```

Comment:
unfortunately these changes surface a bug in name-table - i have a fix for that but if you want I can rollback those changes and provide another PR to fix this

Reply:
the problem is in the offset-computation where there are 2 problems:
* it's not guaranteed that the length of the string has been computed
* the offset computation is done BEFORE the entries are sorted
I think currently one most of the time does not recognize that because if one call normalize() and then getData() the content is correct because on the get data-call

Reply:
Oh if you have a fix for it you can just throw it into this PR. I unfortunately haven't worked on this project in a long time so I'm not really familiar with the specifics here (I also haven't touched java in years either)